### PR TITLE
[Rails5] Fix Line Item wrong type

### PR DESCRIPTION
**Pull request by duduribeiro**
Add an error if the Line Item type is wrong

### CODE REVIEW

File: app/lib/finance/billing.rb [hunk 1]
```diff
@@ -14,6 +14,10 @@ def create_line_item(params)
       li = LineItem.new(params, {without_protection: true})
       li.errors.add(:base, :invalid_invoice_state)
       li
+    rescue ActiveRecord::SubclassNotFound
```
Comment (hallelujah) [inline app/lib/finance/billing.rb]: I think this is because of a migration no?

File: app/lib/finance/billing.rb [hunk 2]
```diff
@@ -14,6 +14,10 @@ def create_line_item(params)
...
+      li = LineItem.new(params.except(:type), {without_protection: true})
+      li.errors.add(:base, :invalid_invoice_state)
```
Comment (guicassolato) [inline app/lib/finance/billing.rb]:
```suggestion
      li.errors.add(:base, : invalid_line_item_type)
```
?

Comment (hallelujah) [reply app/lib/finance/billing.rb]: Not really it is the same as above just removing the `type` in params

**Comment by hallelujah** [comment]
I think you can fix CodeClimate and we are good to go

**Pull request merged**